tools/depends/target/mariadb/04-pthread.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt	2021-06-28 08:00:02.761728364 +0000
++++ b/CMakeLists.txt	2021-06-28 08:00:29.198289581 +0000
+@@ -293,7 +293,7 @@
+     IF(OPENSSL_FOUND)
+       ADD_DEFINITIONS(-DHAVE_OPENSSL -DHAVE_TLS)
+       SET(SSL_SOURCES "${CC_SOURCE_DIR}/libmariadb/secure/openssl.c")
+-      SET(SSL_LIBRARIES ${OPENSSL_SSL_LIBRARY} ${OPENSSL_CRYPTO_LIBRARY})
++      SET(SSL_LIBRARIES ${OPENSSL_SSL_LIBRARY} ${OPENSSL_CRYPTO_LIBRARY} ${_OPENSSL_LDFLAGS_OTHER})
```

Comment:
Is the leading `_` on the variable `${_OPENSSL_LDFLAGS_OTHER}` wanted? I haven't seen a leading underscore used for CMake variables often

Reply:
yes